README.md

```diff
@@ -3,7 +3,34 @@
 This project implements the backend for registry.k8s.io, Kubernetes's container
 image registry.
 
-For more see [cmd/archeio](./cmd/archeio/README.md)
+For more details on the implementation see [cmd/archeio](./cmd/archeio/README.md)
+
+The community deployment configs are documented at in the k8s.io repo with
+the rest of the community infra deployments:
+https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io
+
+## Privacy
+
+This project abides by the Linux Foundation privacy policy, as documented at
+https://registry.k8s.io/privacy
+
+## Background
```

Comment:
this is a much shorter version, to capture some context knowing users have link fatigue, it is not intended to replace the communication about why we made the move wholesale, but to be sufficient that we can point users to this repo with other current details at the `/` redirect

Reply:
(and to consolidate current documentation of what exactly registry.k8s.io _is_, we also now have inbound links from https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io and hopefully soon https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io)